test/e2e/quadlet/name.pod

```diff
@@ -0,0 +1,4 @@
+## assert-podman-pre-args "--name=test-pod"
```

Comment:
Can we also check for `pod create`?

Reply:
The test for `basic.pod` already checks it. The test files used for specific keys check only the key and not entire command to reduce test failure logs